indigo_api/tests/test_importers.py

```diff
@@ -362,6 +362,31 @@ def test_subparagraphs_1(self):
 <p>(iii) item</p>
 <p>(aa) item</p>
 <p>(bb) item</p>
+""").strip())
+
+    def test_paragraphs_with_inline_whitespace_at_start(self):
+        self.assertMultiLineEqual(
+            """
+BODY 
+
+  PARAGRAPH (a)
+
+    chief electoral officer;
+
+  PARAGRAPH (b)
+
+    deputy chief electoral officer; and
+
+  PARAGRAPH (c)
+
+    electoral officer.
+""".strip(),
+
+            self.html_to_bluebell("""
+<p>	(<em>a</em>)	chief electoral officer;</p>
+<p>	(<em>b</em>)	deputy chief electoral officer; and</p>
+<p>
+<em>	</em>(<em>c</em>)	electoral officer.</p>
```

Comment:
Can you also add a case with whitespace before the "c" within the element: `<em>  c</em>`.

Reply:
actually, more like `<p>  <em>   (c)</em> saoeu</p>`

Reply:
Also add one with `<u>` because `RemoveEmptyInlines` removes a, b, i, sup and sub but not u.

Reply:
I can add it; it won't pass as is. We haven't encountered this yet as far as I know?